### ISSUE TITLE
Coral Dev Board

### ISSUE THREAD

**Issue by agherzan** [balena-os/resin-os-device-support#82]
* **CPU**:  NXP i.MX 8M SOC (quad Cortex-A53, Cortex-M4F)
* **GPU**: Integrated GC7000 Lite Graphics
* **Memory**: 1 GB LPDDR4
* **Storage**: 8 GB eMMC 
* **Features**:
  * Wi-Fi 2x2 MIMO (802.11b/g/n/ac 2.4/5GHz) Bluetooth 4.1 
  * Google Edge TPU coprocessor
  *

* **Link**: https://coral.withgoogle.com/products/dev-board/

**Comment by harborretail** [comment]
Any update available on this development?

Very interested in this hardware platform.

**Comment by Jaeyoung-Lim** [comment]
Would be pretty cool to have this in the fleet!

**Comment by slessans** [comment]
Any updates on this?

**Comment by acostach** [comment]
yes @slessans , it's in PR stage, will most likely become available in the staging dashboard in the following days: https://github.com/balena-os/balena-coral/pull/1
